pom.xml

```diff
@@ -569,6 +571,17 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
Maybe add an XML comment stating these are for Netex import.

Reply:
I removed the 'com.migesok:jaxb-java-time-adapters' dependency - it was not used in OTP code, only by the `netex-java-model`. I also added a comment on `netex-java-model`.